airbyte-ci/connectors/pipelines/pipelines/steps/simple_ci_step.py

```diff
@@ -0,0 +1,115 @@
+from pipelines import main_logger
+from dataclasses import dataclass
+from pathlib import Path
+from pipelines.actions.environments import with_installed_pipx_package, with_python_base
+from pipelines.bases import Step, StepResult
+from pipelines.contexts import PipelineContext
+
+import dagger
+
+from typing import List, Optional
+
+
+@dataclass(frozen=True)
+class MountPath:
+    path: Path
+    optional: bool = False
+
+    def __post_init__(self):
+        if not self.path.exists():
+            message = f"{self.path} does not exist."
+            if self.optional:
+                raise FileNotFoundError(message)
+            else:
+                main_logger.warning(message)
+
+    def __str__(self):
+        return str(self.path)
+
+    @property
+    def is_file(self) -> bool:
+        return self.path.is_file()
+
+
+class SimpleCIStep(Step):
```

Comment:
Is there a good way to unit test this new type of step? 🙂 maybe something that runs some echoes of some env vars, mounted paths, etc? I'm reviewing the code but with less familiarity with dagger, test assertions would be helpful I feel